src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -29,65 +25,42 @@
 import javax.servlet.http.HttpServletResponse;
 
 // Servlet that handles comments
-@WebServlet("/make-comment")
+@WebServlet("/comment")
 public class CommentServlet extends HttpServlet {
   /* This method takes comments, timestamps and the id of comments and creates a Task object.
    * It then converts the Task object to a json.
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
-
-    DatastoreService savedComments = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = savedComments.prepare(query);
-
-    List<Task> tasks = new ArrayList<>();
-
-    for (Entity entity : results.asIterable()) {
-      long id = entity.getKey().getId();
-      long timestamp = (long) entity.getProperty("timestamp");
-      String comment = (String) entity.getProperty("comment");
-
-      Task task = new Task(id, timestamp, comment);
-      tasks.add(task);
+    List<Comment> comments;
+    if (getParameter(request, "store", "").isEmpty()) {
+      comments = CommentDatabase.getCommentByEmail(getParameter(request, "email", ""));
+    } else {
+      comments = CommentDatabase.getCommentByStore(getParameter(request, "store", ""));
     }
 
     Gson gson = new Gson();
 
     response.setContentType("application/json;");
-    response.getWriter().println(gson.toJson(tasks));
+    response.getWriter().println(gson.toJson(comments));
   }
 
   // This method takes input from the comment box and stores it with the rest of the comments
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String comment = getParameter(request, "text-input", "");
-    long timestamp = System.currentTimeMillis();
-
-    Entity taskEntity = new Entity("Task");
-    taskEntity.setProperty("comment", comment);
-    taskEntity.setProperty("timestamp", timestamp);
+    String drink = getParameter(request, "drink", "");
+    drink = drink.toLowerCase();
+    long rating = Integer.parseInt(getParameter(request, "rating", ""));
+    String content = getParameter(request, "content", "");
+    String store = getParameter(request, "store", "");
+    String email = getParameter(request, "email", "");
```

Comment:
Can be addressed in a subsequent pull request, but should we allow user comments without a store and email?